app/helpers/await.js

```diff
@@ -1 +1 @@
-export { default, await } from 'ember-promise-helpers/helpers/await';
+export { default, awaitFor } from 'ember-promise-helpers/helpers/await';
```

Comment:
Ah, this was the autogenerated thing from Ember CLI and you were updating it so it wasn't using `async` (which i guess is reserved?). Can you change this line to `export { default } from 'ember-promise-helpers/helpers/await';`?